### PR TITLE
Feature/dont analyze under 100 records

### PR DESCRIPTION
**Pull request by dsmits**

### CODE REVIEW

File: v6_carrier_py/master.py [hunk 1]
```diff
@@ -131,15 +132,15 @@ def fit_pipeline(client: ClientContainerProtocol, data, pipe: Pipeline, features
                         `predict` method.
     :param features: The features that should be used in the fitting of the pipeline.
     :param target: The field that should be used as target for the machine learning algorithm.
-    :param identifying_columns: The identifying fields for joining datasets.
+    :param keys: The identifying fields for joining datasets.
```
Comment (svenvanderburg) [inline v6_carrier_py/master.py]: I agree `identifying_columns` is a bit verbose, but `keys` is so much used in other contexts. Maybe we can name it `on` (like in pandas) or `join_keys`, `on_keys`.

File: v6_carrier_py/master.py [hunk 2]
```diff
@@ -177,6 +180,10 @@ def _combine_all_node_data(client, data, identifying_columns, *args, **kwargs) -
                                               subset=identifying_columns)
     n_dropped_rows = len_before_drop - len(combined_df)
     info(f'Dropped {n_dropped_rows} rows with duplicate identifiers')
+
+    if len(combined_df.index) < MIN_RECORDS:
+        raise Exception(f'Only {len(combined_df.index)} records available for analysis! Privacy is not ensured.')
```
Comment (svenvanderburg) [inline v6_carrier_py/master.py]: - Would be nice to also let the user know the MIN_RECORDS
- Better to raise something more specific than `Exception`, maybe `ValueError`?

**Review by svenvanderburg — CHANGES_REQUESTED** [review]
Please take a look at my comments :)

File: v6_carrier_py/master.py (hunk 1 above)
Comment (dsmits) [reply v6_carrier_py/master.py]: I renamed it to `merge_keys`. In the final product this is going to be gone anyway.

File: v6_carrier_py/master.py (hunk 2 above)
Comment (dsmits) [reply v6_carrier_py/master.py]: Done

**Review by svenvanderburg — APPROVED** [review]

**Pull request merged**